requirements.dev-prod.in

```diff
@@ -2,7 +2,6 @@ black==21.7b0
 click==8.0.1
 flake8==3.9.2
 pytest==6.2.4
-pytest-cov==3.0.0
```

Comment:
oh, did you mean to remove this?  it's fine!

Reply:
Yes, based on your comment.

Reply:
It works without it by using `coverage run` as you said.